setup.py

```diff
@@ -29,7 +29,7 @@ def read(fname):
         ],
     },
     install_requires=[
-        'json-rpc~=1.13.0',
+        'json-rpc>=1.12,<2',
```

Comment:
Should this be `>=1.14`? since the `_get_responses` doesn't take that context argument before `1.14`?

Reply:
Oh you're right! But just confirming the tags in the `json-rpc` repo, would it be `>=1.13`?

Reply:
Thanks for the quick adjust of it!!!!

Reply:
Absolutely! Thanks so much for such a thorough review :)